common/client-core/src/client/real_messages_control/acknowledgement_control/mod.rs

```diff
@@ -98,6 +101,8 @@ impl PendingAcknowledgement {
                 recipient_tag,
                 extra_surb_request,
             },
+            // Custom mix hops are not supported for anonymous messages
+            mix_hops: None,
```

Comment:
I wonder if they should (because in the long run I guess we want to be sending replies with surbs)